doc/tutorials/acrn_on_qemu.rst

```diff
@@ -11,8 +11,8 @@ configuration.
 
```

Comment:
Suggest adding a simple block diagram that shows the ACRN architecture that this tutorial is going to provide.

Reply:
The introduction mentions running a User VM, but there are no instructions here to build or launch a User VM image only the Service VM.

Reply:
> Suggest adding a simple block diagram that shows the ACRN architecture that this tutorial is going to provide. Similar to the diagram in https://projectacrn.github.io/2.7/tutorials/nvmx_virtualization.html.

@NanlinXie Please help to check with Dev team.

Reply:
Since this tutorial is low priority, we can add a block diagram later.